config/initializers/new_framework_defaults_6_0.rb

```diff
@@ -0,0 +1,45 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Now that `config.load_defaults` is set to `6.0` I guess this file can be deleted too?

Reply:
It was actually setting a couple of parameters. I moved them to `application.rb` and deleted the file.